### DISCUSSION TITLE
LURO 5M - not finish circle 5751624

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#6] [Uncalculated rounds]
<img width="829" alt="Snímek obrazovky 2024-09-23 v 14 51 37" src="https://github.com/user-attachments/assets/258c9389-c662-4b83-8efa-2f41dba64050">

**Comment by JackHipeek** [comment]
Also another games.
<img width="1018" alt="Snímek obrazovky 2024-09-23 v 14 53 41" src="https://github.com/user-attachments/assets/1959c23c-7a2e-4ff4-b548-0eaf0ccb69d9">

**Reply by bf-monster** [reply]
Hey, ill take a look, thanks

**Comment by bf-monster** [comment]
shoud be fixed now, thank you
